dynawo/sources/API/JOB/JOBOutputsEntry.cpp

```diff
@@ -120,6 +120,16 @@ OutputsEntry::getCurvesEntry() const {
   return curvesEntry_;
 }
 
+void
+OutputsEntry::setFinalStateValuesEntry(const boost::shared_ptr<FinalStateValuesEntry>& finalStateValuesEntry) {
+  finalStateValuesEntry_ = finalStateValuesEntry;
```

Comment:
Copy is missing in the copy method above

Reply:
Sorry, I don't understand the comment. The method is assigning the shared pointer stored in the `OutputsEntry` for final state values entry, the same way it was done before for logs entry or lost equipment. A copy of the final state values entry received should not be copied, just referenced from the outputs entry.

Reply:
I put the comment here as I can't put it above but my comment is about the method ::copy above in the file